src/extensions/default/bramble/stylesheets/sidebarTheme.css

```diff
@@ -1,68 +1,250 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600');
+
 /* Components to hide */
 #working-set-list-container {
-	display: none !important;
+  display: none !important;
 }
 span#project-title.title {
-	display: none;
+  display: none;
 }
 div.working-set-option-btn[style] {
-	display: none !important;
+  display: none !important;
 }
 
-/* Recoloring */
+/* Dark Theme Color */
+/* TODO - move this to the dark theme CSS later */
+
 #sidebar {
-	background-color: black;
-	text-shadow: none;
-	color: #333333;
+  background-color: black;
+  text-shadow: none;
+}
+
+#project-files-container {
+  padding-left: 0px;
+  font-weight: 400;
+  font-family: "Open Sans", "Helvetica Neue";
+}
+
+#project-files-container ul {
+  padding: 0;
+}
+
+
+/* File tree basic styles */
+
+li.jstree-leaf a span,
+li.jstree-leaf span.extension {
+  font-size: 15px;
+  color: rgba(255,255,255,.5);
+}
+
+li.jstree-closed > a span,
+li.jstree-open > a span {
+  font-size: 15px;
+  color: rgba(255,255,255,.7);
+}
+
+
+/* Overrides not needed margin */
+
+.jstree-brackets li.jstree-leaf,
+.jstree-brackets li.jstree-closed,
+.jstree-brackets li.jstree-open {
+  margin-left: 0;
+}
+
+
+/* Basic formatting of all items in file menu */
+
+li.jstree-open a,
+li.jstree-closed a,
+li.jstree-leaf a {
+  padding: 10px 10px 10px 30px;
+  cursor: pointer !important;
 }
-li.jstree-leaf>a {
-	padding: 5px;
+
+
+/* Folders get bold text treatment */
+
+li.jstree-open > a,
+li.jstree-closed > a {
+  font-weight: 600;
+}
+
+
+/* Selected Item */
+
+li.jstree-leaf a.selected-node,
+li.jstree-leaf a.selected-node span,
+li.jstree-leaf a.selected-node .extension {
+  color: white !important;
+}
+
+
+/* Hover state for all items */
+
+li.jstree-leaf a:not(.selected-node):hover,
+li.jstree-open a:first-of-type:hover,
+li.jstree-closed a:first-of-type:hover
+{
```

Comment:
Let's always do `{` on same line as content, as you have in the rest of the file.  Can you change throughout this file so it's consistent?